experiments/list_unsupported.sh

```diff
@@ -34,8 +34,30 @@ done
 
 echo "$1: Unsupported features\n" > "$file_name".txt
 
+export ADA_INCLUDE_PATH=/opt/gnat/lib/gcc/x86_64-pc-linux-gnu/4.9.4/rts-native/adainclude/
+if [ ! -d "$ADA_INCLUDE_PATH" ]; then
+   echo >&2 "Ada include path does not exists!"
+   exit 4
+fi
+
+PATH=/opt/gnat/bin:${PATH}
```

Comment:
I probably wouldn't explicitly set the path here, but instead just do the test for gnat being on the path and leave it up to the user to change their path if needs be.

Reply:
Done.